dev/integration_tests/deferred_components_test/android/build.gradle

```diff
@@ -0,0 +1,35 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
It looks like that script actually creates both the settings.gradle and build.gradle. Though some of it may be useful, it does generate files that do not implement some of the config needed for deferred components.

Actually, I think I need to go in and add a skip to make sure it doesn't produce incompatible configs for deferred components.

Reply:
The script could skip overriding `settings.gradle`. 

I'm mostly concerned about the lockfiles since those help to keep transient dependencies consistent between builds.

Reply:
Ok, I changed to skip settings.gradle. build.gradle is generated now and lockfiles are in.